### PR TITLE
[s]Valentinestest

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
For test merging, since I can't really test fuly this on my own

### CODE REVIEW

File: code/modules/events/holiday/vday.dm [hunk 1]
```diff
@@ -19,6 +19,47 @@
...
+			valentines -= date
+
+			forge_valentines_objective(L, date)
+
+			forge_valentines_objective(date, L)
+
+
+
+		else
+			L << "<span class='warning'><B>You didn't get a date! They're all having fun without you! you'll show them though...</B></span>"
```
Comment (Xhuis) [inline code/modules/events/holiday/vday.dm]: `boldwarning`

File: code/modules/events/holiday/vday.dm [hunk 2]
```diff
@@ -19,6 +19,47 @@
...
+
+	ticker.mode.traitors |= lover.mind
+	lover.mind.special_role = "valentine"
+
+	var/datum/objective/protect/protect_objective = new /datum/objective/protect
+	protect_objective.owner = lover.mind
+	protect_objective.target = date.mind
+	protect_objective.explanation_text = "Protect [date.real_name], your date."
+	lover.mind.objectives += protect_objective
+	lover << "<span class='warning'><B>You're on a date with [date]! Protect them at all costs. This takes priority over all other loyalties.</B></span>"
```
Comment (Xhuis) [inline code/modules/events/holiday/vday.dm]: See above

**Comment by Xhuis** [comment]
grammar whining is done, yw

**Comment by Cyberboss** [comment]
@Xhuis wrong PR

**Comment by Xhuis** [comment]
no pr is safe

**Comment by KingPhilipIII** [comment]
So. What's a martyr supposed to do to get even with the crew?

**Comment by JJRcop** [comment]
@Xhuis #23639 is the actual PR, this one is just for the test merge and will not actually be merged.

File: code/modules/events/holiday/vday.dm (hunk 1 above)
Comment (Supermichael777) [inline code/modules/events/holiday/vday.dm]: gib gun

**Comment by Supermichael777** [comment]
should probably do /datum/objective/hijack instead. STEAL THE LOVEBOAT!

**Comment by Haevacht** [comment]
rrrrrrrrrrrrrrrrrrEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE.

Shit holiday.

**Pull request closed without merging**